tendrl/ceph_integration/manager/__init__.py

```diff
@@ -0,0 +1,138 @@
+import gevent.event
+import logging
+import signal
+
+from tendrl.ceph_integration import ceph
+from tendrl.ceph_integration.manager.cluster_monitor import ClusterMonitor
+from tendrl.ceph_integration.manager import utils
+from tendrl.ceph_integration.central_store import CephIntegrationEtcdPersister
+from tendrl.commons.manager import Manager
+from tendrl.commons.sds_sync import SdsSyncThread
+
+
+LOG = logging.getLogger(__name__)
+
+
+class CephIntegrationSyncStateThread(SdsSyncThread):
+
+    def __init__(self, manager):
+        super(CephIntegrationSyncStateThread, self).__init__()
+
+        self._manager = manager
```

Comment:
remove manager and complete

https://github.com/Tendrl/node-agent/blob/master/tendrl/node_agent/node_sync/__init__.py#L13